services/web/server/src/simcore_service_webserver/storage_api.py

```diff
@@ -43,6 +48,8 @@ async def copy_data_folders_from_project(
             "nodes_map": nodes_map,
         },
         ssl=False,
+        # NOTE: extends time for copying operation
+        timeout=ClientTimeout(total=10 * 60),
```

Comment:
Please assign this to a constant, also consider bumping it 1h. We use 1h timeout for larger projects in other services.